modules/accounting/includes/functions/transactions.php

```diff
@@ -813,6 +813,10 @@ function erp_acct_generate_pdf( $request, $transaction, $file_name = '', $output
     $date = ! empty( $transaction->trn_date ) ? $transaction->trn_date : $transaction->date;
     $trn_pdf->set_reference( erp_format_date( $date ), __( 'Transaction Date', 'erp' ) );
 
+	// Set Date Due
+	$datedue = ! empty( $transaction->due_date) ? $transaction->due_date : $transaction->datedue;
```

Comment:
I am not sure from where `$transaction->datedue` is generating. can you please describe?

Reply:
This is to fetch the due date from the sales invoice created; however, it seems to overlap with other vouchers as well. This needs to be under 'invoice' type and 'estimate' type.

Reply:
we have due date in `due_date` property but i think we are not using the `datedue` property. i think it's better to use a empty string rather than a nonexistent property like following
`$datedue = ! empty( $transaction->due_date) ? $transaction->due_date : '';`